src/app/events/events.component.html

```diff
@@ -0,0 +1,3 @@
+@for (event of events; track event.name) {
```

Comment:
unsure if unique, but urls will be
```suggestion
@for (event of events; track event.url) {
```